learn/variables_and_types.md

```diff
@@ -0,0 +1,200 @@
+## Variable Definition
```

Comment:
Move this heading to ~ 4 line. Text shouldn't begin from a heading. It's better to write a couple of sentences before 1st heading in the text.

Reply:
Or let's remove it completely since we have a "Variable declaration" heading as  well.